From add6274a6adeecf21117dfe0ac56ae16347f7e89 Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Sun, 28 Jun 2026 21:57:39 +0800
Subject: [PATCH 330/334] AOSCOS: net/phytium: guard "np" variable definition
 with #ifdef CONFIG_OF

The of_match_node macro is no-op if !CONFIG_OF, so with CONFIG_WERROR
the build would fail as nothing uses the "np" variable.

Signed-off-by: Xi Ruoyao <xry111@xry111.site>
---
 drivers/net/ethernet/phytium/phytmac_platform.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/net/ethernet/phytium/phytmac_platform.c b/drivers/net/ethernet/phytium/phytmac_platform.c
index 3724936cc826e..225ba0835df92 100644
--- a/drivers/net/ethernet/phytium/phytmac_platform.c
+++ b/drivers/net/ethernet/phytium/phytmac_platform.c
@@ -168,7 +168,9 @@ static int phytmac_get_phy_mode(struct platform_device *pdev)
 static int phytmac_plat_probe(struct platform_device *pdev)
 {
 	const struct phytmac_config *phytmac_config = &phytium_1p0_config;
+#ifdef CONFIG_OF
 	struct device_node *np;
+#endif
 	struct resource *regs;
 	struct phytmac *pdata;
 	int ret, i;
-- 
2.55.0

